maint/mocked/based-notifyfenced.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2019 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+/*
+ * Intended demo use case:
+ *
+ * - start corosync
+ * - start "./based -N"
+ *   (hint: su -s /bin/sh -c './based -N' hacluster)
+ * - wait a bit (5 < seconds < 20)
+ * - run "stonith admin --list-registered"
```

Comment:
stonith_admin

How is pacemaker-fenced being started?

Reply:
Manually, added ;)